Packs/ThreatGrid/Integrations/ThreatGrid/ThreatGrid.py

```diff
@@ -921,6 +932,49 @@ def search_urls():
     })
 
 
+def submit_urls():
+    """
+    Submit urls for analysis
+    """
+    params = {
+        'api_key': API_KEY,
+        'url': demisto.args().get('url')
+    }
+    r = req('POST', SUB_API + 'samples', params=params)
+    res = r.json()['data']
+    return_results({
+        'Type': entryTypes['note'],
+        'EntryContext': {'ThreatGrid.URLs': res},
+        'HumanReadable': tableToMarkdown('ThreatGrid - URL Submission', res),
+        'ContentsFormat': formats['json'],
+        'Contents': res
+    })
+
+
+def advanced_search():
+    """
+    Search Submissions, URLs, Samples with with query
+    """
+    final_results = []
+    results = {}
+    body = demisto.args().get('query')
+    r = req('POST', USER_API + 'search', body=body)
+    markdown = ''
+    if r.json()['data']['sample']:
+        for record in r.json()['data']['sample']:
+            final_results.append(record)
+        markdown += tableToMarkdown('Threat Grid submission results', final_results)
+        results = CommandResults(
+            readable_output=markdown,
+            outputs_prefix='Threatgrid.SearchResult',
```

Comment:
Notice I added the sub key 'SearchResult', otherwise this will show on the root 'Threatgrid" key which is less comfortable for customers.

Reply:
Yup this this looks better. Thank you Jasmine.